lib/sumup.dart

```diff
@@ -101,6 +101,21 @@ class Sumup {
     return SumupPluginResponse.fromMap(method);
   }
 
+  /// To prepare a SumUp card terminal for checkout, a registered SumUp merchant
+  /// account needs to be logged in and the card terminal will have been already
+  /// setup. Calling prepareForCheckout() before instancing a checkout will
+  /// speed up the checkout time.
+  ///
+  /// Login required.
+  static Future<SumupPluginResponse> prepareForCheckout(
+      [bool? retainBLEConnection]) async {
```

Comment:
I'd make retainBLEConnection a named non nullable parameter with a default of false

Reply:
Sounds good. I will update it